README.md

```diff
@@ -24,9 +24,9 @@ Advanced features of Taichi include [spatially sparse computing](https://taichi.
 ```bash
 python3 -m pip install taichi       # Latest version: 0.5.14
```

Comment:
Please update this to 0.5.15.

Reply:
Btw, did you check `Allow edits from maintainers`? If so I should be able to push to this PR. Or maybe `maintainers` > `write-access members`?